desktop/src/main.rs

```diff
@@ -216,12 +216,29 @@ fn load_movie(url: &Url, opt: &Opt) -> Result<SwfMovie, Error> {
     Ok(movie)
 }
 
+fn get_max_screen_size(event_loop: &EventLoop<RuffleEvent>) -> PhysicalSize<u32> {
+    let mut max_screen_size = (0, 0);
+
+    for monitor in event_loop.available_monitors() {
+        let size = monitor.size();
+        max_screen_size.0 += size.width;
```

Comment:
I think summing all widths and heights isn't the way to go.
Consider I have two monitors side-by-side. Now the max screen height will be calculated as the sum of their heights, which is obviously incorrect. What we really want is the *maximum* of their heights.
If one monitor is on top of the other, then we want the sum of their heights, but the maximum of their widths.

My point is that the calculation must take the monitor positions into account. Fortunately winit provides [`MonitorHandle::position()`](https://docs.rs/winit/latest/winit/monitor/struct.MonitorHandle.html#method.position). 
So I think the best way to use it is to accumulate the minimum and maximum of both X and Y coordinates:
```rs
fn get_max_screen_size(event_loop: &EventLoop<RuffleEvent>) -> PhysicalSize<u32> {
    let mut min_x = 0;
    let mut min_y = 0;
    let mut max_x = 0;
    let mut max_y = 0;
    for monitor in event_loop.available_monitors() {
        let size = monitor.size();
        let position = monitor.position();
        min_x = min_x.min(position.x);
        min_y = min_y.min(position.y);
        max_x = max_x.max(position.x + size.width as i32);
        max_y = max_y.max(position.y + size.height as i32);
    }
    let width = max_x - min_x;
    let height = max_y - min_y;
    (width, height).into()
}
```